### ISSUE TITLE
No me funciona el QPixMap

### ISSUE THREAD

**Issue by camiloEscorza** [IIC2233/Syllabus#288]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [X] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/8.-Reglas-del-foro-(issues)).
* [X] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues).
* [X] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al ayudante de Bienestar o al correo del curso (iic2233@ing.puc.cl).
* [X] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda.

Hola buenas, como dice el título no me funciona el QPixMap al ejecutar el codigo copiado de la parte de ‘imagenes‘  del modulo ‘elementos graficos basicos‘ de la semana 5. No tengo el error de importación, osea que el paquete sí está instalado (al parecer), pero al abrirse la ventana no importa la imagen. 
Reitero que no me aparece ningún error, por lo que no es por paquete, PERO si lo ejecuto a través de la terminal, la que detecta otro python (3.10 en vez de 3.11), no tiene instalado el QPixMap. Al intentar instalarlo, no puedo y al intentar actualizar pip para ver si es un error de pip me sale ERROR: Could not install packages due to an OSError: Cannot move the non-empty directory '/Users/camiloescorza/miniconda3/lib/python3.10/site-packages/pip-23.0.1.dist-info/': Lacking write permission to '/Users/camiloescorza/miniconda3/lib/python3.10/site-packages/pip-23.0.1.dist-info/'.  Yo sé que tengo varias versiones de python3 instaladas, pero no estaba seguro hasta qué punto interactúan entre ellas. Para ese python tengo un problema de paquetes, pero para el que uso en vscode no, ya que instalo pyqt5 en ese y todas las funciones (excepto QPixMap) me funcionan
Tambien hay que considerar que uso un M1 y que tambien pueda tener problemas por ese lado, pero no estoy seguro.
Todas las demás cosas (hasta ahora) me han funcionado bien, pero esto en particular no :(

**Comment by Hernan4444** [comment]
Hola @camiloEscorza 

Partiendo por lo más simple, cuando ejecutas el código desde VSCode, ¿a qué te refieres con "al abrirse la ventana no importa la imagen."? ¿Que se muestra la ventana, pero la imagen no se ve?

Queremos un poquito más de información para poder detectar el problema.

Quedamos atentos!
Saludosヾ(＾-＾)ノ

**Comment by camiloEscorza** [comment]
Sí, la ventana se muestra, y todos los elementos de ella se ven, y de hecho las dimensiones del label se mantienen, pero la imagen no se muestra. Cuando le pongo texto al label y luego le pongo un pixmap, el texto se borra (me imagino que se oculta) y no me muestra nada, es transparente.

**Comment by Hernan4444** [comment]
Yap,

Lo que se me ocurre, es que el problema está con el path del QPixmap del código. En ese código, el path es `ruta_imagen = os.path.join('img', 'python.jpg')`. Esto significa, que **Desde donde estés parado para hacer `python3 main.py`** debe existir una carpeta llamada `img` que tenga adentro `python.jpg`.


Si por ejemplo, abriste VSCode desde la carpeta `"contenidos"` y desde ahí intentas ejecutar un archivo python que está dentro de `"contenidos/semana-05"`. VSCode ejecutará tu python pensando que está dentro de la carpeta `"contenidos"`, y por lo tanto el path `"img/python.jpg"` no te llevará a la imagen que buscas. En ese caso, el path debería ser `"semana-05/img/python.jpg"`. No obstante, esta opción no es muy buena, de tener que editar el path.

Lo óptimo es asegurarte de abrir VSCode en la carpeta donde está el `main.py` que quieras ejecutar. En este caso, abrir VSCode desde la carpeta `"semana-05"`. Otra opción, es que en una carpeta totalmente distinta (llamemosla "ejemplo"), pongas el `main.py` y la carpeta `img` dentro de "ejemplo" y le indicas a VSCode que abra la carpeta "ejemplo", y ahí le pidas ejecutar `main.py`. De este modo, los _paths_ deberían estar bien y podrás observar la imagen.

Cuentanos cómo te va con eso,
Saludosヾ(＾-＾)ノ

**Comment by camiloEscorza** [comment]
me funcionooo :)) graciass
